shared/windows/mkdir.c

```diff
@@ -0,0 +1,20 @@
+#include <windows.h>
+#include <wchar.h>
+
+typedef	unsigned short mode_t;
```

Comment:
Please use space instead of tab

Reply:
Did not notice that guy. Easy fix though.

![sneaky](https://user-images.githubusercontent.com/8355441/79079936-8bac8500-7d01-11ea-88a5-c58980691c56.png)